meta-mbl-bsp-common/recipes-bsp/atf/atf.inc

```diff
@@ -9,14 +9,17 @@ DEPENDS += " openssl-native coreutils-native optee-os u-boot virtual/kernel"
 
 PROVIDES += "virtual/atf"
 
+FILESEXTRAPATHS_prepend:="${THISDIR}/files:"
+
 SRC_URI = " \
-    git://git.linaro.org/landing-teams/working/mbl/arm-trusted-firmware.git;protocol=https;nobranch=1;name=atf \
-    git://github.com/ARMmbed/mbedtls.git;protocol=https;branch=development;name=mbedtls;destsuffix=mbedtls \
+    git://git.linaro.org/landing-teams/working/mbl/arm-trusted-firmware.git;protocol=https;nobranch=1;name=atf	\
+    file://0001-imx-imx6-uboot_ddr_init-Import-u-boot-DDR-init-code.patch					\
```

Comment:
This patch should be added in `meta-freescale-3rdparty-mbl/recipes-bsp/atf/atf-imx6ul-pico-mbl_git.bb` instead. Both the `SRC_URI` and the patch file itself (in `meta-freescale-3rdparty-mbl/recipes-bsp/atf/atf-imx6ul-pico-mbl` directory)